### PR TITLE
Fixed issue which results in incorrect selection of index page

### PR DESCRIPTION
**Pull request by ziafazal**
Fixed issue which results in incorrect selection of index page
Fixed broken unit tests

### CODE REVIEW

**Review by etechdevai — APPROVED** [review]
👍

File: scormxblock/scormxblock.py [hunk 1]
```diff
@@ -221,6 +221,7 @@ def _store_unziped_files_to_s3(self):
         self._delete_existing_files(self._file_storage_path())
         local_path = self.local_storage_path
         file_paths = []
+
```
Comment (zubair-arbi) [inline scormxblock/scormxblock.py]: nit: remove this extra git diff

File: scormxblock/scormxblock.py [hunk 2]
```diff
@@ -309,10 +310,8 @@ def get_context_studio(self):
     def get_context_student(self):
         scorm_file_path = ''
         if self.scorm_file:
-            scheme = 'https' if settings.HTTPS == 'on' else 'http'
-            scorm_file_path = '{}://{}{}'.format(
-                scheme,
-                configuration_helpers.get_value('site_domain', settings.ENV_TOKENS.get('LMS_BASE')),
+            scorm_file_path = '{}{}'.format(
+                configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
```
Comment (zubair-arbi) [inline scormxblock/scormxblock.py:313]: We don't set `LMS_ROOT_URL` in site_configuration, instead we override `LMS_ROOT_URL` via django settings override for all sites (https://edlyio.atlassian.net/wiki/spaces/BM/pages/713785351/Edly+Cloud+Client+Setup+Guide+on+Multisite+Instance). How about we update it like this:
```python
scorm_file_path = '{lms_root_url}{scorm_file_name}'.format(
    lms_root_url=settings.LMS_ROOT_URL,
    scorm_file_name=self.scorm_file,
)
```

**Review by zubair-arbi — COMMENTED** [review]
Changes looks good, just few small suggestions.

File: scormxblock/scormxblock.py (hunk 2 above)
Comment (ziafazal) [reply scormxblock/scormxblock.py:313]: I think we should give preference to site config and fallback to settings if site config does have key. That is standard approach

Comment (zubair-arbi) [reply scormxblock/scormxblock.py:313]: Got it. This change is for OpenedX community.

**Pull request merged**